src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -132,6 +133,12 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
       this.workflowsService.getWorkflowVersionOrcidAuthors(this.workflow.id, this.selectedVersion.id).subscribe((orcidAuthors) => {
         this.authors = [...this.selectedVersion.authors, ...orcidAuthors];
       });
+      if (this.selectedVersion.descriptionSource === 'CUSTOM_README') {
+        const slashIndex = this.selectedVersion.readMePath.slice(0, -1).lastIndexOf('/'); //index of the slash before the readme filename. This allows us to get the path to its parent directory.
```

Comment:
If I understand correctly, the `slice(0, -1)` removes the last character from the string, why do we need to do that?